dbus_service.py

```diff
@@ -171,10 +171,10 @@ def _get_config():
         return config
 
     @staticmethod
-    def get_processed_meter_value(meter_data: dict, value: str, default_value: any, factor: int = 1) -> any:
+    def get_processed_meter_value(meter_data: dict, path_to_value, default_value: any, factor: int = 1) -> any:
```

Comment:
Why did you remove the type?

```suggestion
    def get_processed_meter_value(meter_data: dict, path_to_value: str, default_value: any, factor: int = 1) -> any:
```

Reply:
because this is actually not a string - it is an array of strings. But not always... So the "str" here is more confusing than helping

We should clean this up...